sessionctx/variable/session.go

```diff
@@ -1152,6 +1159,19 @@ func (s *SessionVars) WithdrawAllPreparedStmt() {
 	metrics.PreparedStmtGauge.Set(float64(afterMinus))
 }
 
+// SetTmpSystemVar sets the value of a system variable temporarily
+func (s *SessionVars) SetTmpSystemVar(name string, val string) error {
+	s.tmpSystems[name] = val
+	return nil
+}
+
+// ClearTmpSystemVars clear temporarily system variables.
+func (s *SessionVars) ClearTmpSystemVars() {
+	for name := range s.tmpSystems {
+		delete(s.tmpSystems, name)
+	}
```

Comment:
Just creating a new map may be a better way to clear a map in GO~ https://stackoverflow.com/questions/13812121/how-to-clear-a-map-in-go

Reply:
done